apps/test-suite/tests/Battery.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable prettier/prettier */
+import * as Battery from 'expo-battery';
+
+export const name = 'Battery';
+
+export async function test({ describe, it, expect, jasmine }) {
+    describe(name, () => {
+        describe('Test Getters', () => {
+            it('Gets current battery level', async () => {
+                let batteryLevel;
+                let wasRejected = false;
+                try {
+                    batteryLevel = await Battery.getBatteryLevelAsync();
+                } catch (error) {
+                    wasRejected = true;
+                }
+                expect(batteryLevel).toBeLessThanOrEqual(1);
+                expect(batteryLevel).toBeGreaterThan(0);
+                expect(wasRejected).toBe(false);
```

Comment:
Move that line to the top, so the order of failure is: `if rejected -> error that rejected` and not `if rejected -> error that batteryLevel was supposed to be less than or equal 1`